### PR TITLE
UI refresh

### PR DESCRIPTION
**Pull request by artemstepanenko**

### CODE REVIEW

File: src/App.scss [hunk 1]
```diff
@@ -1 +1,25 @@
+@import "./sass/variables";
 @import "./sass/bootstrap-custom";
+
+.panel-heading {
```
Comment (milenvoutchev) [inline src/App.scss]: Please group to a file. Say `sass/common/panel.scss'

File: src/components/BugsHistory/BugsHistory.scss [hunk 2]
```diff
@@ -0,0 +1,6 @@
+@import 'variables';
+
+.bugs-current {
```
Comment (milenvoutchev) [inline src/components/BugsHistory/BugsHistory.scss]: Do not add widget-specific classes as root-level. Group under `.panel--bugshistory`, e.g.:
```
.panel--bugshistory {
  .bugs-current {
  ...
  }
}
```

File: src/containers/Dashboard.scss [hunk 3]
```diff
@@ -1,9 +1,12 @@
 @import 'variables';
 
 .Dashboard {
+  padding-top: 0;
+  //min-height: calc(100vh);
+}
+
+body {
```
Comment (milenvoutchev) [inline src/containers/Dashboard.scss]: Keep in the widget-related .scss files only the styles, related to the according widget. Therefore, move `body {...}` to `scss/_scaffolding.scss`. 
The name of the file comes from Bootstrap, where the body-related classes are grouped inside an include with the same name (node_modules/bootstrap-sass/assets/stylesheets/bootstrap/_scaffolding.scss)

File: src/components/NavigationBar/NavigationBar.scss [hunk 4]
```diff
@@ -1,9 +1,54 @@
 @import 'variables';
 
 .NavigationBar__refresh {
-  margin-right: 10px;
 }
 
-.NavigationBar.navbar {
-  margin-bottom: $margin;
+.navbar {
```
Comment (milenvoutchev) [inline src/components/NavigationBar/NavigationBar.scss]: Same as https://github.com/rocket-internet-berlin/RocketDashboard/pull/19/files#r119548209

File: src/sass/_variables.scss [hunk 5]
```diff
@@ -1,19 +1,29 @@
-$main-color: #337ab7; // is taken from bootstrap ($brand-primary)
+// colors
+$main-color: #1986ff;
+$default-bg-color: #f1f6f8;
+$rocket-gray: #b7b7b7;
```
Comment (milenvoutchev) [inline src/sass/_variables.scss]: Better use semantic name, e.g. `$panel-heading-color`, `$normal-number-color`.

File: src/App.scss (hunk 1 above)
Comment (artemstepanenko) [inline src/App.scss]: Fixed.

File: src/components/BugsHistory/BugsHistory.scss (hunk 2 above)
Comment (artemstepanenko) [inline src/components/BugsHistory/BugsHistory.scss]: Fixed.

File: src/containers/Dashboard.scss (hunk 3 above)
Comment (artemstepanenko) [inline src/containers/Dashboard.scss]: Fixed.

File: src/components/NavigationBar/NavigationBar.scss (hunk 4 above)
Comment (artemstepanenko) [inline src/components/NavigationBar/NavigationBar.scss]: Fixed.

File: src/sass/_variables.scss (hunk 5 above)
Comment (artemstepanenko) [inline src/sass/_variables.scss]: Actually, I like semantic names better. But @gkatsanos said he prefers the way it is now. Could we talk about pros and cons?

**Pull request merged**